src/dummy-data/team.js

```diff
@@ -2,29 +2,55 @@ import Status from '../helpers/Status';
 import { getTeamMembersByKey } from '../raw-data/preprocess_data';
 import { coursesByTeamId } from '../raw-data/preprocess_data';
 import { raw_erteam } from '../raw-data/raw_erteam';
+import { personById } from './person';
 
-function fillTeamList(teamList) {
+
+
+function createTeamList() {
+  const teamList = []
   for (let i in raw_erteam) {
     const teamData = raw_erteam[i];
     // At the moment only return team member's emp ids from first shift
     const memberList = getTeamMembersByKey(teamData.er_team_id, 0).map(member => member.emp_id);
     // Get required courses for team
     const competenceList = coursesByTeamId[teamData.er_team_id];
 
+    let status = Status.OK;
+    // Find team status based on status of members courses for this team
+    for (let j in memberList) { //Check all members
+      const person = personById[memberList[j]];
+
+      for (let k in competenceList) { // Check all required courses
+        const course_id = competenceList[k]
+        // If some person lacks required competence
+        if (person.competence[course_id].status === Status.ERROR ) {

```

Comment:
`person.competence[course_id]` can be undefined for a person that has never completed a course, it just might not be in the current data set.. This line would fail in that case. Do we need to handle that condition?

Reply:
added check for undefined or null values